.github/workflows/go.yml

```
@@ -0,0 +1,27 @@
+name: Go
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      relativePath: ./
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '^1.19'
```

```suggestion
          go-version: '^1.22'
```
https://endoflife.date/go